dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -50,22 +46,20 @@ commons-lang3/3.12.0//commons-lang3-3.12.0.jar
 commons-logging/1.1.3//commons-logging-1.1.3.jar
 commons-math3/3.4.1//commons-math3-3.4.1.jar
 commons-net/3.1//commons-net-3.1.jar
-commons-pool/1.5.4//commons-pool-1.5.4.jar
 commons-text/1.6//commons-text-1.6.jar
 compress-lzf/1.0.3//compress-lzf-1.0.3.jar
 core/1.1.2//core-1.1.2.jar
 curator-client/2.7.1//curator-client-2.7.1.jar
 curator-framework/2.7.1//curator-framework-2.7.1.jar
 curator-recipes/2.7.1//curator-recipes-2.7.1.jar
-datanucleus-api-jdo/4.2.4//datanucleus-api-jdo-4.2.4.jar
```

Comment:
I'm not sure why these datanucleus jars are removed here - they are still used by the `hive` module and present in the Spark distribution jars.